gradle.properties

```diff
@@ -8,4 +8,4 @@ mavenPassword=YourPassword
 # - here-naksha-lib-core/INaksha (static property: latest)
 # - here-naksha-lib-psql/resources/naksha_plpgsql.sql (method: naksha_version)
 # - here-naksha-app-service/src/main/resources/swagger/openapi.yaml (info.version property)
-version=2.0.11
+version=2.0.12
```

Comment:
What's our policy regarding versioning? When do we update the version?
Are we following [semantic versioning](https://semver.org/) rules?